src/client/automation/playback/visible-element-automation.js

```diff
@@ -58,9 +59,20 @@ export default class VisibleElementAutomation extends serviceUtils.EventEmitter
             .then(() => delay(this.automationSettings.mouseActionStepDelay));
     }
 
+    _getElementOffset () {
+        const defaultOffsets = getOffsetOptions(this.element);
+
+        let { offsetX, offsetY } = this.options;
```

Comment:
we can use `let { offsetX = defaultOffsets.offsetX , offsetY = defaultOffsets.offsetY} = this.options;`